frontend/src/shared/layout/notification/notification.tsx

```diff
@@ -31,7 +31,7 @@ function NotificationBanner () {
     const dispatch = useDispatch();
     const notificationService = NotificationService(dispatch);
 
-    useEffect(() => {
+    useMemo(() => {
```

Comment:
I actually think `useEffect` is more appropriate since you're not using the output. But I don't think it makes a difference.

Reply:
I'm using useMemo since it runs on first page load rather than running as a side effect of the page loading. It helps me fix some null errors when passing in components to notifications props.content